README.md

```diff
@@ -6,10 +6,83 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for operating and monitoring electrical
```

Comment:
"The Frequenz Microgrid API is a service"...

Reply:
Done